dateparser/date.py

```diff
@@ -14,6 +14,12 @@
 from dateparser.languages.detection import AutoDetectLanguage, ExactLanguages
 
 
+APOSTROPHE_LOOK_ALIKE_CHARS = [
+    u'\u2019', u'\u02bc', u'\u02bb', u'\u055a', u'\ua78c',
+    u'\uff07', u'\u2032', u'\u2035', u'\u02b9', u'\uff07',
```

Comment:
My position is - take some of both worlds, so either use literals and leave their UTF codes in the comment, or the other way around.
Literals are useful if one is searching in rendered page, UTF code is useful if one is checking code output, HTML etc.
In the future we would move those to separate YML file.